packages/app-desktop/gui/JoplinCloudConfigScreen.scss

```diff
@@ -9,4 +9,25 @@
 	p {
 		padding: calc(var(--joplin-font-size) * 0.8);
 	}
+}
+
+.joplin-cloud-account-information {
+	margin-bottom: calc(var(--joplin-font-size) * 3);
+
+	table {
+		margin-bottom: calc(var(--joplin-font-size) * 1.5);
+		border: none;
```

Comment:
Is it possible to avoid these CSS calculations? As I think that will make it difficult to maintain a consistent style throughout the application. We have the `--joplin-margin` - could that be enough? Or if something else is needed maybe introduce a new margin?